mobify/progressive-web-scaffold
# Enable node modules caching for web.

When we moved Progressive Web to the `web` folder in #214, CircleCI stopped caching the `node_modules` folder in the `web` directory.

## Changes
- Manually added `web/node_modules` to the directories being cached between builds.

## How to test-drive this PR
- Navigate to https://circleci.com/gh/mobify/progressive-web-scaffold/tree/fix-caching.
  - Verify that the first build took significantly longer to complete than subsequent builds.
